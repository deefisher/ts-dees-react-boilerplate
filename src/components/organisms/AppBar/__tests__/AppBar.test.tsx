import { render } from '@testing-library/react';
import { AppBar } from '..';
import { Grommet, Page, Text } from 'grommet';

describe('AppBar', () => {
    it('renders AppBar unchanged', () => {
        const { container } = render(
            <Grommet>
                <Page>
                    {/* <AppBar> */}
                    <Text size="large">My App</Text>
                    {/* </AppBar> */}
                </Page>
            </Grommet>,
        );
        expect(container).toMatchSnapshot();
    });
});
