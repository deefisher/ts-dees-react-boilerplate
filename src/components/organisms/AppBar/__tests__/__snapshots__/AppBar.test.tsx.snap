// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBar renders AppBar unchanged 1`] = `
<div>
  <div
    class="StyledGrommet-sc-19lkkz7-0 ioQEen"
  >
    <div
      class="StyledBox-sc-13pk1d4-0 bDSydy"
    >
      <span
        class="StyledText-sc-1sadyjn-0 fBGZaZ"
      >
        My App
      </span>
    </div>
  </div>
</div>
`;
